frontend/src/pages/UserSettingsPage.js

```diff
@@ -1,13 +1,17 @@
 import React, { useState, useRef } from 'react';
-import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
 
 import '../styles/components/UserSettings.css';
 
 import { Avatar, Button, Card } from '../common';
 import { ProfilePicCropperModal, UserSettingUpdateModal } from '../components';
 import { usersData } from '../data';
+import { toggleTheme as DarkLightTheme } from '../actions/local';
+
+function UserSettingsPage() {
+  const isDarkTheme = useSelector((state) => state.local.darkTheme);
+  const toggleTheme = useDispatch();
```

Comment:
Usually you'd call this dispatch instead of toggleTheme

Reply:
Correct, I keep consistency as **`toggleTheme`** variable use at bottom. We will change when other actions use in this component.